### ISSUE TITLE
Unknown image block appears when dragging blocks in the List View

### ISSUE THREAD

**Issue by exstheme** [WordPress/gutenberg#37806]
### Description

When dragging blocks in the List view - unknown image block is created with error message.

### Step-by-step reproduction instructions

1. Create new post
2. Create two paragraphs 
3. Open list view
4. Drag paragraphs
5. Unknown new image block appears

### Screenshots, screen recording, code snippet

[https://cdn.exsthemewp.com/gutenberg_bug.gif](https://cdn.exsthemewp.com/gutenberg_bug.gif)

### Environment info

WordPress 5.9RC1
Version 96.0.4664.93 (Official Build) for Linux Mint (64-bit)

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by talldan** [comment]
That's curious! I tried testing following the instructions (using Mac OS / Brave) and couldn't reproduce the issue.

@exstheme Which browser are you using?

**Comment by exstheme** [comment]
Sorry, forgot to add browser name into Environment info section. I'm using Chromium browser for development and testing purposes.

I have recently tested list view dragging in the Firefox and the official Chrome browser and it seems that there is no same issue in these browsers. Only in the Chromium web browser. It is really strange. My Chrome browser has same build version as Chromium:

Google Chrome
Version 96.0.4664.93 (Official Build) (64-bit)

UPD:

My Brave browser also have this issue:

https://cdn.exsthemewp.com/gutenberg_bug_brave.gif

**Comment by annezazu** [comment]
Added as browser issues. I couldn't replicate in the Site Editor or Post Editor with 5.9 RC1 on Chrome or Safari.

**Comment by piotrbak** [comment]
Also not able to reproduce this on Chrome.

**Comment by monika-12** [comment]
Couldn't replicate in the Post Editor with 5.9 RC2 on Chrome.

**Comment by alanjacobmathew** [comment]
Couldn't replicate on Brave and Edge browser . 5.9 RC2

**Comment by annezazu** [comment]
@exstheme can you ensure all browser add-ons are disabled as well? Want to fully rule everything out :) Otherwise, likely going to need to close this out since others are unable to replicate.

**Comment by exstheme** [comment]
Hello, @annezazu !

I have tried to use dragging in the List view in the Incognito window and everything worked as expected.
Than I have closed all of my tabs in my browsers, relaunched them and then tested once again - and there was no same issue after that. Even  with all of my browser's extensions enabled.

It seems that this is some kind of issue when many browser tabs are opened or something like this.

Thanks everyone for testing!
